src/explore-education-statistics-common/src/modules/charts/components/__tests__/MapBlockInternal.test.tsx

```diff
@@ -15,7 +17,7 @@ import produce from 'immer';
 import React from 'react';
 import userEvent from '@testing-library/user-event';
 
-describe('MapBlock', () => {
+describe('MapBlockInternal', () => {
```

Comment:
I've moved the tests to be on MapBlockInternal as that's where the code actually is and I've seen an intermittent failure on the first test where it wasn't rendering the map. Probably a long shot but maybe the tests have problems with the dynamic import of MapBlockInternal so have bypassed that.